daemons/execd/remoted_tls.c

```diff
@@ -38,12 +42,169 @@ gnutls_dh_params_t dh_params;
 static int ssock = -1;
 extern int lrmd_call_id;
 
+static bool try_getting_schema_files = true;
+
 static void
 debug_log(int level, const char *str)
 {
     fputs(str, stderr);
 }
 
+static void
+write_extra_schema_file(xmlNode *xml, void *user_data)
+{
+    const char *file = NULL;
+    char *path = NULL;
+    int fd;
+    int rc;
+
+    file = crm_element_value(xml, PCMK__XA_PATH);
+    if (file == NULL) {
+        return;
+    }
+
+    /* If the referenced path includes a '/', build up the expected directory
+     * structure under PCMK__REMOTE_SCHEMA_DIR.
+     */
+    if (strchr(file, '/') != NULL) {
```

Comment:
Yeah, we don't actually need to copy that one over. It doesn't hurt but it might drag in a bunch of api/ unnecessarily. I've been wanting to move the CIB files to an xml/cib/ subdirectory anyway (first part of https://projects.clusterlabs.org/T8 ) so maybe this is a good time to do it.

Reply:
Right, we don't need to copy that one, and I think that is the only href that uses a subdirectory.  Nothing should pull in api/ because nothing in the cib-related files references anything in it.  I'm fine with removing all this subdirectory crud as long as we can remember the rule that schema files can't use subdirectories.

Reply:
Removed.